### ISSUE TITLE
table-group: HTML comments

### ISSUE THREAD

**Issue by jacobq** [ember-template-lint/ember-template-lint#553]
I realize that [HTML comments are frowned upon](https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/rules/lint-no-html-comments.js), but it still seems like a bug that the `table-groups` rule will trigger ("error  Tables must have a table group (thead, tbody or tfoot).  table-groups") on comments.

*Fails*
```html
<table><!-- this --></table>
<table>{{! or this }}</table>
```
*Passes*
```html
<table></table>
```

This may relate to these:
https://github.com/ember-template-lint/ember-template-lint/issues/285
https://github.com/ember-template-lint/ember-template-lint/issues/464

**Comment by rwjblue** [comment]
Definitely open to fixing this (we should allow both HTML comments and handlebars comments)...

**Comment by rwjblue** [comment]
Fixed by https://github.com/ember-template-lint/ember-template-lint/pull/638
